qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -224,7 +234,9 @@ def run_test(self):
         # If we attempt to spend with the default privacy policy, z_sendmany
         # returns an error because it needs to create a transparent recipient in
         # a transaction involving a Unified Address.
-        revealed_recipients_msg = "This transaction would have transparent recipients, which is not enabled by default because it will publicly reveal transaction recipients and amounts. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` parameter set to `AllowRevealedRecipients` or weaker if you wish to allow this transaction to proceed anyway."
+        revealed_recipients_msg = """This transaction cannot be sent as is. The following issues were identified:
+* This transaction would have transparent recipients, which requires AllowRevealedRecipients because it will publicly reveal amounts and addresses of the transaparent funds.
```

Comment:
```suggestion
* This transaction would have transparent recipients, which requires AllowRevealedRecipients because it will publicly reveal amounts and addresses of the transparent funds.
```